codalab/lib/bundle_cli.py

```diff
@@ -1297,7 +1266,7 @@ def do_edit_image_command(self, args):
 
         print >>self.stdout, '===='
         print >>self.stdout, 'Entering container {}'.format(container_id[:8])
-        print >>self.stdout, 'Once you are happy with the changes, please exit the container (ctrl-D)'
+        print >>self.stdout, 'If you would like to commit the changes, please exit the container (ctrl-D)'
         print >>self.stdout, 'and commit your changes to a new image by running:'
         print >>self.stdout, ''
         print >>self.stdout, '\tcl commit-image {} [image-tag]'.format(container_id[:8])
```

Comment:
Discourage this behavior.  Instead ask them to create a Dockerfile (point to the Wiki).